### PR TITLE
[IT] Add README, index.md, and week01/01.md

### PR DESCRIPTION
**Pull request by edran**
As per PR title, this creates the basic files for the Italian translation + gets `week01.01` done :)

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -123,3 +123,9 @@ es:
     - path: es/week01/01.md
       sections:
         - path: es/week01/01-1.md
+
+it:
+  title:   'Apprendimento Profondo'
+  tagline: 'Gotta learn them all in Italian'
```
Comment (ebetica) [inline docs/_config.yml]: This was actually useless, can you delete it?

Comment (edran) [reply docs/_config.yml]: Ooooh you are the webmaster for the course! Excellent to be working with you again 💃 

Will do :)

Comment (ebetica) [reply docs/_config.yml]: :dancer:

Comment (Atcold) [reply docs/_config.yml]: What's going on here? You know each other?!

Comment (edran) [reply docs/_config.yml]: Alright, removed -- @ebetica where / how are you translating the title now for each language?

Comment (Atcold) [reply docs/_config.yml]: Just the `tagline`, not everything! Haha!

Comment (Atcold) [reply docs/_config.yml]: Going to cook some lunch, then I'll merge this!
I can't wait to have an Italian version up!

Comment (edran) [reply docs/_config.yml]: >Just the tagline, not everything! Haha!

Oh noes -- sigh -- I'll revert the reverting.

>What's going on here? You know each other?!

Fellow dancers of the tensors 💃(ask Zeming directly :P )

@Atcold hold up merging / reviewing this, let me clean it up first :)

**Comment by edran** [comment]
@Atcold @ebetica plz review now

File: docs/it/index.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
+---
+layout: default
+title: APPRENDIMENTO PROFONDO
+author: Alfredo Canziani
+lang-ref: home
+lang: it
+---
```
Comment (Atcold) [inline docs/it/index.md]: Add
```
translation-date:
translator:
```

File: docs/it/index.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+|Grader|<img src="https://st3.depositphotos.com/13159112/17145/v/450/depositphotos_171453724-stock-illustration-default-avatar-profile-icon-grey.jpg" width="100" height="100">|Raghav Jajodia <br>rj1408@nyu.edu|
+|Grader|<img src="https://st3.depositphotos.com/13159112/17145/v/450/depositphotos_171453724-stock-illustration-default-avatar-profile-icon-grey.jpg" width="100" height="100">|Priyank Pathak <br>pp1953@nyu.edu|
+|Grader|<img src="https://st3.depositphotos.com/13159112/17145/v/450/depositphotos_171453724-stock-illustration-default-avatar-profile-icon-grey.jpg" width="100" height="100">|Chiao-Hsun Wang <br>chw371@nyu.edu|
+|Grader|<img src="https://st3.depositphotos.com/13159112/17145/v/450/depositphotos_171453724-stock-illustration-default-avatar-profile-icon-grey.jpg" width="100" height="100">|Pedro Vidal<br>pmh314@nyu.edu|
+|Grader|<img src="https://st3.depositphotos.com/13159112/17145/v/450/depositphotos_171453724-stock-illustration-default-avatar-profile-icon-grey.jpg" width="100" height="100">|Bixing Yan <br>by783@nyu.edu|
```
Comment (Atcold) [inline docs/it/index.md]:
```suggestion
```

**Comment by edran** [comment]
Done -- btw @ebetica I've seen some inconsistency wrt `translator:` 1-2 people have put down their GitHub username rather than their real name.

Probably not that urgent, but might be worth having everyone use their GH username, and pull the real names (if they exist) at build time via GH's API?

File: docs/it/index.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
+---
+layout: default
+title: APPRENDIMENTO PROFONDO
```
Comment (Atcold) [inline docs/it/index.md]:
```suggestion
title: Apprendimento Profondo
```

**Comment by Atcold** [comment]
Feedback:
 1. Rimuovi `it/` dai link, che altrimenti puntano a `it/it`. 
 2. Taglia i nomi sulla homepage (non devono essere esaustivi / corretti). Altrimenti da cellulare viene una porcata.

Screenshot 2020-03-27 22 47 19

**Comment by Atcold** [comment]
"SGD e retroprop" può bastare, ad esempio.

**Pull request merged**